news/2021-04-08-monthly-beatmapping-contest-april-2021.md

```diff
@@ -0,0 +1,111 @@
+
+**[osu!taiko MBC](https://osu.ppy.sh/community/contests/123) limitation:**
+
+- No more than 3 of the same note colours in a row.
+
+![](/wiki/shared/news/2021-04-08-monthly-beatmapping-contest-april-2021/colours.png)
+
+**[osu!catch MBC](https://osu.ppy.sh/community/contests/124) limitation:**
+
+- Sliders cannot be used.
+
+### <a id="songs"></a>Songs
+
+Each MBC provides beatmappers a choice between songs from one of osu!'s [Featured Artists](https://osu.ppy.sh/beatmaps/artists).
+
+This month's artist is the vocal-chopping expert [**MYUKKE.**](https://osu.ppy.sh/beatmaps/artists/121). These three songs are available for mapping in the osu!, osu!taiko, and osu!catch contests:
+
+#### Boudica (165 BPM)
+
+[Beatmap template download](https://assets.ppy.sh/artists/121/WE%20LOVE%20MYUKKE.%20EP/MYUKKE.%20-%20Boudica.osz) (2:15)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/121/WE%20LOVE%20MYUKKE.%20EP/MYUKKE.%20-%20Boudica.mp3" type="audio/mpeg">
+</audio>
+
+#### Ginevra (170 BPM)
+
+[Beatmap template download](https://assets.ppy.sh/artists/121/Songs/MYUKKE.%20-%20Ginevra.osz) (1:57)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/121/Songs/MYUKKE.%20-%20Ginevra.mp3" type="audio/mpeg">
+</audio>
+
+#### OCCHOCO-REST-LESS (135 BPM)
+
+[Beatmap template download](https://assets.ppy.sh/artists/121/Songs/MYUKKE.%20-%20OCCHOCO-REST-LESS.osz) (2:07)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/121/Songs/MYUKKE.%20-%20OCCHOCO-REST-LESS.mp3" type="audio/mpeg">
+</audio>
+
+## Rules
+
+You might have limitless mapping ideas, but you'll still need to fit within the bounds of the contest to participate:
+
+- **follow the month's [beatmapping limitation](#limitation) for the contest's mode:**
+    - **osu!:**
+        - All sliders must use red anchors.
+        - Red anchors must form clear corners in the slidershape.
+    - **osu!taiko:**
+        - No more than 3 of the same note colours in a row.
+    - **osu!catch:**
+        - Sliders cannot be used.
+- **be in `.osz` format.** Including a background image and custom hitsounds is fine, but storyboard/video content will not be considered.
+- **be a single difficulty for one of the month's [selected songs](#songs).**
+- **not be uploaded to the osu! website through the Beatmap Submission System (*BSS*).** All contest submissions are anonymous during the screening/judging phases to avoid bias. **DO NOT SHARE YOUR ENTRY!**
+- **be solely created by you.** Collaborations are not allowed.
+- **abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
```

Comment:
Only if the phrase `The submission must` placed before each criterion, will these bullet points make sense. I suggest that you should either apply the following or add the phrase above all companied with a comma/colon/semi-colon.
```suggestion
- **The submission must follow the month's [beatmapping limitation](#limitation) for the contest's mode:**
    - **osu!:**
        - All sliders must use red anchors.
        - Red anchors must form clear corners in the slidershape.
    - **osu!taiko:**
        - No more than 3 of the same note colours in a row.
    - **osu!catch:**
        - Sliders cannot be used.
- **The submission must be in `.osz` format.** Including a background image and custom hitsounds is fine, but storyboard/video content will not be considered.
- **The submission must be a single difficulty for one of the month's [selected songs](#songs).**
- **not be uploaded to the osu! website through the Beatmap Submission System (*BSS*).** All contest submissions are anonymous during the screening/judging phases to avoid bias. **DO NOT SHARE YOUR ENTRY!**
- **The submission must be solely created by you.** Collaborations are not allowed.
- **The submission must abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
```